translation/lint.rst

```diff
@@ -32,3 +32,16 @@ The linter results can be exported to JSON using the ``--format`` option:
 
     $ php bin/console lint:yaml translations/ --format=json
     $ php bin/console lint:xliff translations/ --format=json
+
+.. tip::
+
+    The Yaml component provides a stand-alone ``yaml-lint`` binary to lint YAML
+    files without using the entire Symfony console application:
```

Comment:
console component?

Reply:
That wouldn't be precise because you still need the Console component to run this binary. But I agree that it's a bit tricky to explain, so we can improve the description.

Reply:
Most of the time the application used is the one from framework-bundle not the console component,  I'm fine with the current description though.